.editorconfig

```diff
@@ -194,7 +194,7 @@ csharp_preserve_single_line_blocks = true
 csharp_preserve_single_line_statements = false
 
 # C# formatting settings - Namespace options
-csharp_style_namespace_declarations = file_scoped:suggestion
+csharp_style_namespace_declarations = file_scoped:error
```

Comment:
Only change besides a single command in VS to fix all IDE0161 (file-scoped namespace suggestion) in the solution.